.eslintrc

```diff
@@ -31,10 +31,11 @@
     ],
     "no-underscore-dangle": 0,
     "no-undef": 2,
-    "no-unused-vars": 0,
+    "no-unused-vars": 1,
     "quotes": [
       2,
       "single"
-    ]
+    ],
+    "semi": 1
   }
-}
\ No newline at end of file
+}
```

Comment:
This is alright, but next time pls separate lint fixes and feature changes into separate commits